analysis/paladinholy/src/modules/shadowlands/conduits/UntemperedDedication.tsx

```diff
@@ -0,0 +1,60 @@
+import SPELLS from 'common/SPELLS';
+import conduits from 'common/SPELLS/shadowlands/conduits';
+import SpellLink from 'interface/SpellLink';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import { RemoveBuffEvent } from 'parser/core/Events';
+import SUGGESTION_IMPORTANCE from 'parser/core/ISSUE_IMPORTANCE';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+
+class UntemperedDedication extends Analyzer {
+  // a counter for when UD stacks are dropped
+  private totalDrops = 0;
+  conduitRank = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasConduitBySpellID(SPELLS.UNTEMPERED_DEDICATION.id);
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.UNTEMPERED_DEDICATION.id);
+    if (!this.active) {
+      return;
+    }
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.UNTEMPERED_DEDICATION_BUFF),
+      this.onByPlayerUntemperedDedicationBuffRemoval,
+    );
+  }
+
+  onByPlayerUntemperedDedicationBuffRemoval(event: RemoveBuffEvent) {
+    this.totalDrops += 1;
+  }
+
+  get dropSuggestion() {
+    return {
+      actual: this.totalDrops,
+      isGreaterThan: {
+        minor: 1,
+        average: 2,
+        major: 3,
+      },
+      style: ThresholdStyle.NUMBER,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.dropSuggestion).addSuggestion((suggest, actual) =>
+      suggest(
+        <span>
+          You dropped your <SpellLink id={SPELLS.UNTEMPERED_DEDICATION.id} /> stacks {actual} times.
+          Try to keep it up at all times, as this is a fairly large loss of healing.
+        </span>,
+      )
+        .icon(conduits.UNTEMPERED_DEDICATION.icon)
+        .staticImportance(SUGGESTION_IMPORTANCE.MAJOR)
```

Comment:
This shouldn't be here. Like at all i'm pretty sure

This basically said `if there is a problem. make it major` Which this metric is very prone to things like boss downtime meaning making it major always feels wrong